services/docker/docker-size.service.js

```diff
@@ -12,6 +12,7 @@ import {
 const buildSchema = Joi.object({
   name: Joi.string().required(),
   full_size: nonNegativeInteger.required(),
+  images: Joi.any().required(),
```

Comment:
We shouldn't be using `any()`. The point of defining schemas is to make sure all the values we need are in our input object and handle it cleanly if they are not. `any()` will pass any old value which could lead to us throwing unhandled runtime exceptions later in the code. There are more docs on how and why we validate at https://github.com/badges/shields/blob/master/doc/input-validation.md

Reply:
Good point, thanks, updated.